packages/app/i18n/en.pot

```diff
@@ -459,16 +456,10 @@ msgid ""
 "you may not have access to it."
 msgstr ""
 
-msgid "Series is empty"
+msgid "{{visType}} is empty"
```

Comment:
Looks like these didn't get re-generated?

Reply:
It's weird how it doesn't regenerate all the time, had to delete the file and rebuild for it to work. Fixed.